### ISSUE TITLE
Detect indent

### ISSUE THREAD

**Issue by brandonkal** [Alhadis/Unfuck-Tabstops#1]
It would be nice if this code detected indent size. i.e. Code that is indented with four spaces would be replaced with a tab character. This would allow running the bookmarklet automatically regardless of the source.

**Comment by Alhadis** [comment]
Yeah, there's a number of things I've yet to fix, but haven't gotten around to doing so yet. This is one of them.
